app/controllers/donations.js

```diff
@@ -140,6 +140,10 @@ module.exports = (app) => {
                 }
               };
 
+              if (process.env.NODE_ENV === 'production') {
+                charge.application_fee_percent = constants.OC_FEE_PERCENT;
```

Comment:
This isn't going to work. Charge is declared below.

Also, this means that our platformFee amount will never get tested in non-production. It's going to break other tests as well. 

I think a better way might be to create a new entry in local .env file:

```
STRIPE_SANITIZE_KEY: "xxxxx"
```
You can get this key by authorizing stripe on your local machine. Then, update the sanitize script to use this key.